docs/azure/vscodeforweb.md

```diff
@@ -155,13 +179,13 @@ If you encounter any issues with VS Code for the Web – Azure, please log an is
 
 ### Account type prompt issue
 
-If you see a prompt asking "What type of account did you use to start this tunnel?" then it's likely that you're on vscode.dev/azure and not insiders.vscode.dev/azure. Currently this product is only available on Insiders. To fix this, navigate to <https://insiders.vscode.dev/azure>.
+If you see a prompt asking "What type of account did you use to start this tunnel?" then it's likely that you're on vscode.dev/azure and not vscode.dev/azure. To fix this, navigate to <https://vscode.dev/azure>.
```

Comment:
This is confusing because you mention the same link twice, namely vscode.dev/azure. Should one of these be insiders.vscode.dev/azure?